### ISSUE TITLE
Cannot operate CMD.EXE

### ISSUE THREAD

**Issue by deadlyBuzz** [conda/conda#10664]
## Current Behavior

I was going through an Anaconda tutorial on Udemy and after attempting to set up my Virtual Environment, started getting the error: 

Failed to create temp directory "c:\users\<name>\appData\local\temp\conda-<RANDOM>\"

I have recently uninstalled anaconda using (Add/remove programs) due to the size of the package and my command prompt will no longer work.
While debugging, I opened powershell as administrator and was presented with the message 

& : The term 'C:\Alan\Python\Anaconda3\Scripts\conda.exe' is not recognized as the name of a cmdlet, function, script
file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct
and try again.
At C:\Users\Alan Curley\Documents\WindowsPowerShell\profile.ps1:4 char:4

* (& "C:\Alan\Python\Anaconda3\Scripts\conda.exe" "shell.powershell" "h ...
*    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  * CategoryInfo          : ObjectNotFound: (C:\Alan\Python\...ripts\conda.exe:String) [], CommandNotFoundException
  * FullyQualifiedErrorId : CommandNotFoundException

There is clearly something left behind from the anaconda install that I need to remove.

### Steps to Reproduce

```
See above,
```

## Expected Behavior

I should be able to run my powershell without anaconda environment issues

**Comment by deadlyBuzz** [comment]
I've since managed to remove this message by removing the data from the profile.ps1 file above which contained:

\#region conda initialize
\# !! Contents within this block are managed by 'conda init' !!
\(& "C:\Alan\Python\Anaconda3\Scripts\conda.exe" "shell.powershell" "hook") | Out-String | Invoke-Expression
\#endregion

**Comment by richy330** [comment]
Had the same issue, thanks for sharing your solution. Maybe the removal of this file could be included automatically when uninstalling Anaconda?

**Comment by wulmer** [comment]
I have prepared a PR which adds a check to this command whether conda is still installed... see https://github.com/conda/conda/pull/11374 if you are interested. However the developers seem to be busy at the moment, let's see if I get this PR to be merged soon.

**Comment by erjenkins29** [comment]
Adding a note here in the hope to prioritize this issue -- I spent several hours today trying to figure out why my cmd.exe (and a lot of other programs dependent on a functioning cmd.exe) were failing.  The cause was that I had uninstalled and removed conda from my machine days before, and without specifically gearing my research toward this fact, I likely never would have found the cause of the issue.  Not good.  This article was where my resolution was found : https://www.winhelponline.com/blog/cant-open-cmd-after-uninstalling-python-anaconda/
